imports/blocks/add-to-cart/Subfund/__tests__/Layout.js

```diff
@@ -0,0 +1,121 @@
+
+import { mount } from "enzyme";
+import { mountToJson } from "enzyme-to-json";
+import { reset, startBuffering } from "aphrodite/lib/inject";
+import { getSingleSpecWrapper } from "../../../../util/tests/data-spec.js";
+
+import Layout from "../Layout";
+
+const generateComponent = (additionalProps={}) => {
```

Comment:
change spacing to:
`const generateComponent = (additionalProps = {}) => {`

The spacing around the `=`